src/main/routes/health.ts

```diff
@@ -24,11 +48,13 @@ export default function (app: Application): void {
             return false;
           });
       }),
+      'pcs-api': healthcheck.web(apiHealthUrl, healthOptions('Health check failed on pcs-api:')),
     },
     readinessChecks: {
       shutdownCheck: healthcheck.raw(() => {
         return shutdownCheck() ? healthcheck.down() : healthcheck.up();
       }),
+      'pcs-api': healthcheck.web(apiReadinessUrl, healthOptions('Readiness check failed on pcs-api:')),
```

Comment:
please remove from readiness. It might not be a hard dependency.

Reply:
updated